app-ios/Modules/Sources/About/AboutView.swift

```diff
@@ -122,14 +122,17 @@ public struct AboutView<ContributorView: View, StaffView: View, SponsorView: Vie
                     .padding(.vertical, 24)
 
                     Text("アプリバージョン")
-                        .font(Font.system(size: 14, weight: .medium))
+                        .font(Font.custom(FontAssets.Montserrat.medium, size: 14))
+                        .fontWeight(.medium)
```

Comment:
[Q]
`FontAssets.Montserrat.medium` seems a medium-sized font, so is this modifier redundant? 👀

Reply:
@tkhs0604 
I was using the original description as is, so I certainly don't need this modifier.
I have removed it!🫡